markdown/3.x/en/api/ff-checkout-tracking.api.md

```diff
@@ -6,10 +6,17 @@ ___
 | **disable-auto-tracking**&nbsp;(Boolean) (default: false) | Controls whether the element sends a tracking request upon its rendering. |
 | **records**&nbsp;(Array) | The records returned from FACT-Finder. |
 
+### Methods
+| Name | Description |
+| ---- | ----------- |
+| **trackCheckoutItems** | Dispatches a tracking checkout event for each of its `ff-checkout-tracking-item` children. |
+
+
 ## `ff-checkout-tracking-item`
 ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
 | **record-id**&nbsp;(String) (default: empty) | The item's ID used for tracking in FACT-Finder. |
 | **count**&nbsp;(String) (default: "1") | Number of pieces of this item that were bought. |
+| **price**&nbsp;(String) | Price of the bought item. If omitted it will be inferred from the clicked item and the configured [fieldRoles](/documentation/3.x/field-roles). |
```

Comment:
A comma please.
```suggestion
| **price**&nbsp;(String) | Price of the bought item. If omitted, it will be inferred from the clicked item and the configured [fieldRoles](/documentation/3.x/field-roles). |
```